src/SocialAuthUserManager.php

```diff
@@ -168,14 +168,14 @@ public function setSessionKeysToNullify(array $session_keys) {
    * @return \Symfony\Component\HttpFoundation\RedirectResponse
    *   A redirect response.
    */
-  public function authenticateUser($name, $email, $pluginId, $provider_user_id, $picture_url = FALSE, $data = '') {
+  public function authenticateUser($name, $email, $provider_user_id, $token, $picture_url = FALSE, $data = '') {
 
     // Checks for record in social _auth entity.
-    $user_exist = $this->checkIfUserExists($pluginId, $provider_user_id);
+    $user_exist = $this->checkIfUserExists($provider_user_id);
 
     // Checks if user has authenticated role and no record exist.
     if ($this->currentUser->isAuthenticated() && !$user_exist) {
-      return $this->addUserRecord($this->currentUser->id(), $pluginId, $provider_user_id, $data);
+      return $this->addUserRecord($this->currentUser->id(), $this->pluginId, $provider_user_id, $token, $data);
```

Comment:
You're still passing the unnecessary argument $pluginId here and in other statements